pipelines/nextflow/workflows/additional_seq_prepare/main.nf

```diff
@@ -23,6 +23,8 @@ params.accession = null
 params.prefix = null
 params.PROD_NAME = null
 
+// Optional params
+params.cache_dir = "./cache"
```

Comment:
Shouldn't this be defined in `nextflow.config` instead?

Reply:
sure